include/dlaf/lapack/gpu/lacpy.h

```diff
@@ -0,0 +1,36 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#ifdef DLAF_WITH_CUDA
+
+#include <cublas_v2.h>
+#include <cuda_runtime.h>
+
+#include "dlaf/types.h"
+
+namespace dlaf::gpulapack {
+
+template <class T>
+void lacpy(cublasFillMode_t uplo, SizeType m, SizeType n, const T* a, SizeType lda, T* b, SizeType ldb,
```

Comment:
With the aim of unifying our API, probably it would be better to use `blaspp::Uplo` instead of `cublasFillMode_t`, and convert it internally.